x/ibc/20-transfer/keeper/callbacks_test.go

```diff
@@ -1,11 +1,7 @@
 package keeper_test
 
-import (
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	channel "github.com/cosmos/cosmos-sdk/x/ibc/04-channel"
-	"github.com/cosmos/cosmos-sdk/x/ibc/20-transfer/types"
-)
-
+// XXX: move to 04-channel after CheckOpen
```

Comment:
address

Reply:
`CheckOpen` is not implemented in this PR, channel handshake related message types will be added in a separated PR, renamed XXX to TODO

Reply:
Is there an open issue for this work? Sounds like this would also be blocking for GoZ.